app/models/manageiq/providers/embedded_ansible/crud_common.rb

```diff
@@ -66,6 +66,11 @@ def send_notification(op_type, manager_id, params, success)
         }
       )
     end
+
+    # Override in subclass if necessary
+    def notification_excludes
+      [:name, :manager_ref]
```

Comment:
As per my comment above, here we can just do `[:name, :manager_ref] + Authentication.encrypted_columns`, which I think covers everythings.

Reply:
See my response to your other comment.